spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../lib/player'
+require_relative '../lib/game'
+require_relative '../lib/board'
+require_relative '../lib/modules/helper'
+
+describe Game do
+  let(:player_one) { Player.new('Fred', 'x') }
+  let(:player_two) { Player.new('Alex', 'o') }
+  let(:game) { Game.new(player_one, player_two) }
+  let(:game_class) { Game }
+
+  describe '#choose_signs' do
+    it 'returns an array with the random signs' do
+      expect(game_class.choose_signs).to be_an(Array)
+    end
+  end
+
+  describe '#validate_name' do
+    it 'returns false only when name is short' do
+      expect(game_class.validate_name('Alan')).to_not be_falsy
+    end
+  end
```

Comment:
It seems that this test is not enough to completely test the `validate_method` 👇 . I have changed the method but the tests are still passing. So you need to test the logic from all angles (name greater than or equal to 3 and less than 3). You will notice that this will cover both the positive and negative scenarios.

![image](https://user-images.githubusercontent.com/57726348/101669796-62065200-3a78-11eb-8352-9fe3517fd312.png)

 - [ ] Please add more tests for this method.